### ISSUE TITLE
Problem with neurite

### ISSUE THREAD

**Issue by h-mayorquin** [ClandininLab/bifrost#1]
The current instructions for running the pipeline fail as it is because neurite is using a deprecated command of the inspect library:

https://github.com/adalca/neurite/pull/75

Was fixed with this commit
https://github.com/adalca/neurite/commit/abb5b5059175cbd8382a8f8373c3a7909053b4a6

But they don't seem to have an structured policy of releases:
https://pypi.org/project/neurite/#history

Maybe you want to pin that dependency so your fresh installation works with your demo dataset

**Comment by rueberger** [comment]
Good catch. I've pinned forks of all of the voxelmorph dependencies in `setup.py` for precisely this reason. How did you install bifrost?

**Comment by h-mayorquin** [comment]
pip install -e . from the git local clone.
